docs/spec.rst

```diff
@@ -149,6 +149,19 @@ Following mapping keys are supported (all but the marked once are optional):
         The path in which ``python setup.py install`` will be executed.
         Defaults to the repository root.
 
+    ``version``
```

Comment:
This could be moved into docs on the actual config object, as it's not important for the spec. None of the elements have an example currently, so I don't think one is necessarily warranted here yet. I could see overhauling this doc to include more information eventually however.